plugins/omazureeventhubs/omazureeventhubs.c

```diff
@@ -514,66 +505,59 @@ writeProton(instanceData *__restrict__ const pData,
 		tzParamStrLen,
 		(const char*)pData->amqp_address));
 	// Add to helper Array
-	pData->aProtonMsgs[iMsg] = fmsgEntry;
+	pWrkrData->aProtonMsgs[iMsg] = fmsgEntry;
 finalize_it:
 	RETiRet;
 }
 
 BEGINdoHUP
 CODESTARTdoHUP
-DBGPRINTF("doHUP: ENTER\n");
-#if 0
-// TODO CHECK?!
-	if (pData->bReopenOnHup) {
-		CHKiRet(setupKafkaHandle(pData, 1));
-	} else {
-		/* Optional */
-		const int callbacksCalled = rd_kafka_poll(pData->rk, 0); /* call callbacks */
-		LogMsg(0, NO_ERRCODE, LOG_INFO, "omazureeventhubs: doHUP kafka - '%s' outqueue length: %d,"
-			"callbacks called %d\n", pData->tplName,
-			rd_kafka_outq_len(pData->rk), callbacksCalled);
-	}
-finalize_it:
-#endif
+	// Nothing todo in doHUP for the time being
```

Comment:
IIRC this entry point can be removed if not required (framework will check that) - 85% sure ;-)

Reply:
allright will remove it next amend